### PR TITLE
Improve cassandra MAM backend performance

### PR DESCRIPTION
**Pull request by rslota**
This PR changes cassandra's client library to cqerl which is better supported by community then seestar. Also there are some changes in cassandra DB schema in order to improve query performance.

### CODE REVIEW

File: apps/ejabberd/src/mongoose_cassandra_worker.erl [hunk 1]
```diff
@@ -9,34 +9,14 @@
...
--export([prepared_queries/0]).
+-export([start_link/1]).
 
 %% gen_server callbacks
 -export([init/1, handle_call/3, handle_cast/2, handle_info/2,
          terminate/2, code_change/3]).
+-export([queue_length/1]).
 
 -behaviour(gen_server).
 -behaviour(mongoose_cassandra).
```
Comment (gadgetci) [inline apps/ejabberd/src/mongoose_cassandra_worker.erl]: According to **Compiler**:
> Warning: undefined callback function prepared_queries/0 (behaviour 'mongoose_cassandra')

File: .travis.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 language: erlang
+dist: trusty
```
Comment (michalwski) [inline .travis.yml]: Why do we need this?

Comment (rslota) [inline .travis.yml]: Presets from "matrix" by default run on 12.04 which is unable to compile one of the new dependencies due to lack of C++11 support in GCC.

Comment (michalwski) [inline .travis.yml]: I see. Sounds like a bug in travis though.

**Comment by rslota** [comment]
For just the Cassandra client change, please see #1145 .

**Comment by michalwski** [comment]
Thanks!

**Pull request merged**